<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketOrderDlg</class>
 <widget class="QDialog" name="TicketOrderDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ticket order</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="customer_label">
     <property name="text">
      <string>&amp;Customer</string>
     </property>
     <property name="buddy">
      <cstring>customer_line_edit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QLineEdit" name="customer_line_edit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="when_label">
     <property name="text">
      <string>&amp;When</string>
     </property>
     <property name="buddy">
      <cstring>dateTimeEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>123</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="price_label">
     <property name="text">
      <string>$Price</string>
     </property>
     <property name="buddy">
      <cstring>price_spin_box</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="price_spin_box">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="prefix">
      <string>$</string>
     </property>
     <property name="maximum">
      <double>5000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="quantity_label">
     <property name="text">
      <string>Quantity</string>
     </property>
     <property name="buddy">
      <cstring>quantity_spin_box</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="quantity_spin_box">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="amount_label">
     <property name="text">
      <string>Amount</string>
     </property>
     <property name="buddy">
      <cstring>amount_line_edit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLineEdit" name="amount_line_edit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>$ 0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>375</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2" colspan="4">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>customer_line_edit</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>price_spin_box</tabstop>
  <tabstop>quantity_spin_box</tabstop>
  <tabstop>amount_line_edit</tabstop>
  <tabstop>button_box</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>TicketOrderDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>TicketOrderDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
